core/src/test/java/org/mule/el/datetime/DateTimeTimeTestCase.java

```diff
@@ -80,118 +84,112 @@ public void format()
         df.setTimeZone(TimeZone.getTimeZone("UTC"));
-        Assert.assertEquals(df.format(now.toDate()), now.format("hh:mm:ss"));
+        assertThat(now.format("hh:mm:ss"), is(df.format(calendarNow.getTime())));
     }
 
     @Test
     public void timeZone()
     {
-        Assert.assertEquals(TimeZone.getTimeZone("UTC").getDisplayName(), now.getTimeZone());
+        assertThat(now.getTimeZone(), is(TimeZone.getTimeZone("UTC").getDisplayName()));
     }
 
     @Test
     public void plusSeconds()
     {
-        Assert.assertEquals((Calendar.getInstance().get(Calendar.SECOND) + 1) % 60, now.plusSeconds(1)
-            .getSeconds());
+        assertThat(now.plusSeconds(1).getSeconds(), is((Calendar.getInstance().get(Calendar.SECOND) + 1) % 60));
     }
 
     @Test
     public void plusSecondsRollover()
     {
-        Assert.assertEquals(1, now.plusHours(172800).toCalendar().get(Calendar.DAY_OF_YEAR));
+        assertThat(now.getTime().plusHours(172800).toCalendar().get(Calendar.DAY_OF_YEAR), is(1));
     }
 
     @Test
     public void plusMinutes()
     {
-        Assert.assertEquals((Calendar.getInstance().get(Calendar.MINUTE) + 1) % 60, now.plusMinutes(1)
-            .getMinutes());
+        assertThat(now.getTime().plusMinutes(1).getMinutes(), is((calendarNow.get(Calendar.MINUTE) + 1) % 60));
     }
 
     @Test
     public void plusMinutesRollover()
     {
-        Assert.assertEquals(1, now.plusHours(2880).toCalendar().get(Calendar.DAY_OF_YEAR));
+        assertThat(now.getTime().plusHours(2880).toCalendar().get(Calendar.DAY_OF_YEAR), is(1));
     }
 
     @Test
     public void plusHours()
     {
-        Assert.assertEquals(
-            (Calendar.getInstance(TimeZone.getTimeZone("UTC")).get(Calendar.HOUR_OF_DAY) + 1) % 24,
-            now.plusHours(1).getHours());
+        assertThat(now.getTime().plusHours(1).getHours(), is((calendarNow.get(Calendar.HOUR_OF_DAY) + 1) % 24));
     }
 
     @Test
     public void plusHoursRollover()
     {
-        Assert.assertEquals(1, now.plusHours(48).toCalendar().get(Calendar.DAY_OF_YEAR));
+        assertThat(now.getTime().plusHours(48).toCalendar().get(Calendar.DAY_OF_YEAR), is(1));
     }
 
     @Test
     public void withTimeZone()
     {
         int hour = now.getHours();
-        assertEquals(hour, now.withTimeZone("GMT-03:00").getHours());
+        assertThat(now.withTimeZone("GMT-03:00").getHours(), is(hour));
     }
 
     @Test
     public void changeTimeZone()
     {
         int hour = now.getHours();
-        assertEquals((hour + 24 - 3) % 24, now.changeTimeZone("GMT-03:00").getHours());
+        assertThat(now.changeTimeZone("GMT-03:00").getHours(), is((hour + 24 - 3) % 24));
     }
 
     @Test
     public void changeTimeZoneRollover()
     {
-        Assert.assertEquals(1, now.plusHours(48).toCalendar().get(Calendar.DAY_OF_YEAR));
+        assertThat(now.getTime().plusHours(48).toCalendar().get(Calendar.DAY_OF_YEAR), is(1));
```

Comment:
I'm not getting this test... the name refers to change the timezone, but it compares DayOfYear.